borg/archiver.py

```diff
@@ -1076,6 +1107,34 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='REPOSITORY', nargs='?', default='',
                                type=location_validator(archive=False))
 
+        subparser = subparsers.add_parser('key-export', parents=[common_parser],
```

Comment:
We already have `borg help <thing>`. Same semantics...

I'm not really inclined to add something to 1.0-maint that we already know we want to do different in non-compatible ways. Since this is something supported by argparse out-of-the-box I see no technical issues; and the diff between the two solutions will likely be <10 lines.